.github/scripts/pytest_md_summary.py

```diff
@@ -39,19 +39,19 @@ def parse_xml_report(xml_file) -> None:
         time_duration = float(testcase.get("time", "0"))
         message = ""
         if testcase.find("failure") is not None:
-            status = "$${\color{red}Failed}$$"
```

Comment:
Have you tried html tag?
<code style="color : name_color">text</code>

Reply:
<code style="color : green">text</code> 

it does not work